t/mock.t

```diff
@@ -1417,7 +1417,199 @@ GET /t
 --- udp_query eval
 "\x{00}}\x{01}\x{00}\x{00}\x{01}\x{00}\x{00}\x{00}\x{00}\x{00}\x{00}\x{03}www\x{06}google\x{03}com\x{00}\x{00}\x{10}\x{00}\x{01}"
 --- response_body
-records: [{"class":1,"name":"www.google.com","ttl":123456,"txt":["hello","world!"],"type":16},{"class":1,"name":"www.google.com","ttl":123456,"txt":"blah","type":16}]
+records: [{"class":1,"name":"www.google.com","section":1,"ttl":123456,"txt":["hello","world!"],"type":16},{"class":1,"name":"www.google.com","section":1,"ttl":123456,"txt":"blah","type":16}]
+--- no_error_log
+[error]
+
```

Comment:
Please use the `reindex` script to re-format the `.t` test files you have edited:

https://raw.githubusercontent.com/agentzh/old-openresty/master/bin/reindex

Thanks!